scripts/switch.py

```diff
@@ -152,6 +206,12 @@ def _init_cutout(self):
 # https://github.com/keyboardio/keyswitch_documentation/blob/master/datasheets/ALPS/SKCL.pdf
 class SwitchAlpsMatias(Switch):
 
+    switch_cut_w = switch_w = 15.5
+    switch_cut_h = switch_h = 12.8
+    pin_1_pos = vector(-2.5, -4)
+    pin_2_pos = vector(2.5, -4.5)
```

Comment:
I wasn't sure on this. Maybe an iterable, maybe a dict. If I was doing something more complex I'd also want to think about how best to stare associated data like the hotswap offsets. I left it at this because it seemed like a good-enough solution that leaves options open.